packages/expect/src/jest-expect.ts

```diff
@@ -699,6 +699,82 @@ export const JestChaiExpect: ChaiPlugin = (chai, utils) => {
       )
     },
   )
+
+  /**
+   * Used for `toHaveBeenCalledBefore` and `toHaveBeenCalledAfter` to determine if the expected spy was called before the result spy.
+   */
+  function predicate(expectInvocationCallOrder: number[], resultInvocationCallOrder: number[], failIfNoFirstInvocation: number): boolean {
```

Comment:
I think the name is too generic for this giant file

Reply:
Indeed but should I find a better name or is there a file where I could put this function?

Reply:
Let's find a better name. The refactoring should be done separetly